adafruit_lsm303_accel.py

```diff
@@ -333,64 +340,66 @@ def _lsb_shift(self):  # pylint:disable=too-many-branches
 
         if lsb is -1:
             raise AttributeError(
-                "'impossible' range or mode detected: range: %d mode: %d"
-                % (self._cached_range, self._cached_mode)
+                f"""'impossible' range or mode detected:
+                range: {self._cached_range} mode: {self._cached_mode}"""
```

Comment:
I believe this will add the newline character, so let's do:

```suggestion
                "''impossible' range or mode detected: "
                f"range: {self.cached_range} mode: {self._cached_mode}"
```

Reply:
yeah, I almost went that way at first